### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sudeepa88**
It would be enter for windows users.

for windows users it will be a problem.

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/a_first_splash/index.md [hunk 1]
```diff
@@ -487,7 +487,7 @@ Let's play with some browser objects a bit.
 
    The `value` property represents the current value entered into the text field. You'll see that by entering this command, we've changed the text in the text field!
 
-5. Now try typing `guesses` into the console and pressing return. The console shows you that the variable contains a {{htmlelement("p")}} element.
+5. Now try typing `guesses` into the console and pressing enter. The console shows you that the variable contains a {{htmlelement("p")}} element.
```
Comment (estelle) [inline files/en-us/learn/javascript/first_steps/a_first_splash/index.md:490]: Would this be more inclusive?

```suggestion
5. Now try typing `guesses` into the console and pressing <kbd>enter</kbd> or <kbd>return</kbd>. The console shows you that the variable contains a {{htmlelement("p")}} element.
```

Comment (Josh-Cena) [reply files/en-us/learn/javascript/first_steps/a_first_splash/index.md:490]: See also https://github.com/mdn/content/pull/27473. I would recommend the same here: `Enter` by default, with a caveat that it's called `Return` sometimes.

```suggestion
5. Now try typing `guesses` into the console and pressing <kbd>Enter</kbd> (or <kbd>Return</kbd>, depending on your keyboard). The console shows you that the variable contains a {{htmlelement("p")}} element.
```

Comment (sudeepa88) [reply files/en-us/learn/javascript/first_steps/a_first_splash/index.md:490]: That's better if we change it like that return or enter.

**Comment by Josh-Cena** [comment]
LGTM; letting @estelle take another look

**Review by estelle — APPROVED** [review]
Thanks! 🎉
 Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Comment by sudeepa88** [comment]
Thank You so much 🤗

**Pull request merged**